common/src/main/java/com/graphicsfuzz/common/util/SameValueRandom.java

```diff
@@ -20,10 +20,16 @@
 
   private final boolean boolValue;
   private final int intValue;
+  private final long longValue;
 
-  public SameValueRandom(boolean boolValue, int intValue) {
+  public SameValueRandom(boolean boolValue, int intValue, long longValue) {
     this.boolValue = boolValue;
     this.intValue = intValue;
+    this.longValue = longValue;
+  }
+
+  public SameValueRandom(boolean boolValue, int intValue) {
+    this(boolValue, intValue, 0L);
   }
```

Comment:
Please get rid of this, and instead adapt each existing SameValueRandom constructor to take 0L for its third argument.